src/fswAlgorithms/effectorInterfaces/solarArrayReference/solarArrayReference.h

```diff
@@ -25,29 +25,43 @@
 #include "cMsgCInterface/NavAttMsg_C.h"
 #include "cMsgCInterface/AttRefMsg_C.h"
 #include "cMsgCInterface/HingedRigidBodyMsg_C.h"
+#include "cMsgCInterface/VehicleConfigMsg_C.h"
+#include "cMsgCInterface/RWSpeedMsg_C.h"
+#include "cMsgCInterface/RWArrayConfigMsg_C.h"
 
 
-enum attitudeFrame{
+typedef enum attitudeFrame{
     referenceFrame = 0,
     bodyFrame = 1
-};
+} AttitudeFrame;
+
+typedef enum pointingMode{
+    powerGeneration = 0,
+    momentumDumping  = 1
```

Comment:
This enum variable `momentumDumping` is repeated in `fswAlgorithms/effectorInterfaces/thrusterPlatformReference/thrusterPlatformReference`.  This is causing this RST documentation error:

```
/Users/hp/Repos/basilisk/docs/source/Documentation/fswAlgorithms/effectorInterfaces/thrusterPlatformReference/thrusterPlatformReference.rst:132: WARNING: Duplicate C++ declaration, also defined at Documentation/fswAlgorithms/effectorInterfaces/solarArrayReference/solarArrayReference:145.
Declaration is '.. cpp:enum:: momentumDumping'.
```

Can we give this a unique name, like `momentumManagement`?

Reply:
Done